core/platform/models_test.py

```diff
@@ -0,0 +1,281 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests interface for storage model switching."""
+
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+
+class RegistryUnitTest(test_utils.GenericTestBase):
+    """Tests the Registry class interface."""
+
+    def setUp(self):
+        super(RegistryUnitTest, self).setUp()
+        self.registry_instance = models.Registry()
+
+    def test_import_models_activity(self):
+        """Tests import_models function with activity option."""
+        from core.storage.activity import gae_models as activity_models
+        expected_activity_model = [activity_models]
+        self.assertEqual(
+            tuple(expected_activity_model),
+            self.registry_instance.import_models([models.NAMES.activity]))
+
+    def test_import_models_audit(self):
+        """Tests import_models function with audit option."""
+        from core.storage.audit import gae_models as audit_models
+        expected_audit_model = [audit_models]
+        self.assertEqual(
+            tuple(expected_audit_model),
+            self.registry_instance.import_models([models.NAMES.audit]))
+
+    def test_import_models_base_model(self):
+        """Tests import_models function with base model option."""
+        from core.storage.base_model import gae_models as base_models
+        expected_base_models = []
```

Comment:
Here and elsewhere, I'm not sure why we need to create an empty list, append an entry to it, and then change it into a tuple. Why don't we just declare the tuple directly from the get-go in the self.assertEqual statement, or alternatively have a single line saying `expected_audit_models = (audit_models,)`?